### PR TITLE
Add "check-scripts" to make.ps1

### PR DESCRIPTION
**Pull request by abcdefg30**
For windows users. :)

### CODE REVIEW

File: make.ps1 [hunk 1]
```diff
@@ -177,6 +178,19 @@ elseif ($command -eq "check")
 		UtilityNotFound
 	}
 }
+elseif ($command -eq "check-scripts")
+{
+	foreach ($script in ls "mods/*/maps/*/*.lua")
+	{
+		luac $script
```
Comment (Mailaender) [inline make.ps1]: Try `luac -p`.

File: make.ps1 [hunk 2]
```diff
@@ -177,6 +178,19 @@ elseif ($command -eq "check")
...
+	}
+	foreach ($script in ls "lua/*.lua")
+	{
+		luac $script
+	}
+	rm luac.out
```
Comment (Mailaender) [inline make.ps1]: That avoids the actual compilation and does only a syntax check.

Comment (abcdefg30) [inline make.ps1]: This was just to remove the `luac.out` file generated by the check, but the `-p` flag does this better/anyway.

**Comment by abcdefg30** [comment]
Updated.

**Pull request merged**